SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -1,136 +1,69 @@
+#### Using Receptionist - Local Deployments
+The Receptionist is a SpatialOS service which allows you to connect to a deployment using the host IP and port of the SpatialOS deployment. You can specify these parameters in command line arguments when booting a client and you will connect automatically via receptionist. An example of the receptionist being used to automatically connect a client on boot can be found in the `LaunchClient.bat` script found in our [UnrealGDKExampleProject](https://github.com/spatialos/UnrealGDKExampleProject).
+
+To connect to a deployment using `ClientTravel` and the receptionist flow, simply call `APlayerController::ClientTravel` with the receptionist IP of the host machine and the port the SpatialOS deployment is running on. For example:
 
 ```
-FString TravelURL = TEXT("127.0.0.1:7777/DestinationMap");
+FString TravelURL = TEXT("127.0.0.1:7777");
 PlayerController->ClientTravel(TravelURL, TRAVEL_Absolute, false /*bSeamless*/);
 ```
 
-#### Using Locator
+#### Using Locator - Cloud Deployments
 The Locator is a SpatialOS service which allows you to connect to cloud deployments. 
 
-> Experimental: You can use `ClientTravel` to change which cloud deployment a client-worker is connected to. This functionality is highly experimental and requires you to write your own authorization code.
+> Experimental: You can use `ClientTravel` to change which cloud deployment a client is connected to. This functionality is experimental and requires you to write your own authorization code.
 
-Using the locator flow is very similar to using the receptionist, except with different URL options. You must add the `locator`, or `legacylocator` option, and specify the appropriate options.  The `locator` workflow makes use of the new [Authentication flow](https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication-platform-sdk), and the `legacylocator` makes use of the [Deprecated Authentication flow](https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication)
+Using the locator flow is very similar to using the receptionist, except with different URL options. You must add the `locator` option, and specify the appropriate options.  The `locator` workflow makes use of the new [Authentication flow](https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication-platform-sdk).
 
-**Locator**: Add the options `locator`, `playeridentity` and `login`. For more information about the options, see [Map travel URL options]({{urlRoot}}/content/command-line-arguments#map-travel-url-options).
+**Locator**: Add the options `locator`, `playerIdentityToken` and `loginToken`. For more information about the options, see [Connection Arguments]({{urlRoot}}/content/command-line-arguments#connection-arguments).
```

Comment:
Were these options changed? Last I checked they were still `playeridentity` and `login`

Reply:
Looked at the ConnectionConfig and this is what they were, unless the options are different to what's in the WorkerSDK

Reply:
Actually it seems like the commandline params are not used at all and they must be URL params which are the way you suggested but in the SpatialNetDriver. We need to clean this up but will make the docs change now.